test/functional/apps/discover/_large_string.ts

```diff
@@ -35,12 +35,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await security.testUser.setRoles(['kibana_admin', 'kibana_large_strings']);
       await esArchiver.load('empty_kibana');
       await esArchiver.loadIfNeeded('hamlet');
-      await kibanaServer.uiSettings.replace({ defaultIndex: 'testlargestring' });
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: 'testlargestring',
+        'discover:searchFieldsFromSource': false,
```

Comment:
Think it would be cleaner not to enable the `discover:searchFieldsFromSource` for this single functional, since this is 'experimental'

Reply:
I had already changed the test to pass with the fields API 😂 This was an easier fix. But can revert it to the old behavior